datasets/comonscreens.yaml

```diff
@@ -0,0 +1,31 @@
+Name: Common Screens
+Description: A corpus of web screenshot and metadata data composed of over 70 million websites.
+Documentation: https://commonscreens.com/?page_id=1492
+Contact: admin@commonscreens.com
+ManagedBy: "[Common Screens](https://commonscreens.com/)"
+UpdateFrequency: Monthly
+Tags:
+  - aws-pds
+  - encyclopedic
+  - natural language processing
+  - internet
+License: This data is available for anyone to use subject to rules and regulations of USA.
+Resources:
+  - Description: Common Screens (jpeg and csv format)
+    ARN: arn:aws:s3:::common-screens
+    Region: us-west-2
+    Type: S3 Bucket
+    AccountRequired: True
+DataAtWork:
+  Tutorials:
+    - Title: Example Search Engine

```

Comment:
Please remove Example Search Engine and Similar Site Finder as they do not look like Tutorials?